reconcile/quay_mirror.py

```diff
@@ -261,26 +281,21 @@ def process_sync_tasks(self):
 
         return sync_tasks
 
-    def _is_deep_sync(self, interval):
-        control_file_name = "qontract-reconcile-quay-mirror.timestamp"
-        control_file_path = os.path.join(tempfile.gettempdir(), control_file_name)
+    def _check_compare_tags_elapsed_time(self, interval: int) -> bool:
```

Comment:
Isn't that a good candidate for a unit test? :)

Reply:
heheh, yes, it is... I will add them :)

Reply:
fixed